carbonmark/components/pages/Project/Purchase/PurchaseInputs.tsx

```diff
@@ -0,0 +1,186 @@
+                value: 1,
+                message: t({
+                  id: "purchase.input.amount.minimum",
+                  message: "The minimum amount to buy is 1 Tonne",
+                }),
+              },
+              max: {
+                value: Number(formatUnits(props.listing.leftToSell)),
+                message: t({
+                  id: "purchase.input.amount.maxAmount",
+                  message: "You exceeded the available amount of tonnes",
+                }),
+              },
+            }),
+          }}
+          label={t`How many tonnes of carbon do you want to buy?`}
+          errorMessage={formState.errors.amount?.message}
+          hideLabel
+        />
+
+        <div className={styles.paymentLabel}>
+          <Text>{t`Pay with:`}</Text>
+          <Text t="body3">
+            {t`Balance:`}{" "}
+            {!props.balance ? (
+              <i>{t`Loading...`}</i>
+            ) : (
+              formatToPrice(props.balance, locale)
+            )}
+          </Text>
+        </div>
+
+        <Dropdown
+          name="paymentMethod"
+          initial={carbonmarkPaymentMethodMap["usdc"].id}
+          className={styles.paymentDropdown}
+          aria-label={t`Toggle payment method`}
+          renderLabel={(selected) => (
+            <div className={styles.paymentDropDownHeader}>
+              <Image
+                className="icon"
+                src={
+                  carbonmarkPaymentMethodMap[
+                    selected.id as CarbonmarkPaymentMethod
+                  ].icon
+                }
+                width={28}
+                height={28}
+                alt={
+                  carbonmarkPaymentMethodMap[
+                    selected.id as CarbonmarkPaymentMethod
+                  ].id
+                }
+              />{" "}
+              {selected.label}
+            </div>
+          )}
+          control={control}
+          options={Object.values(carbonmarkPaymentMethodMap).map((val) => ({
+            id: val.id,
+            label: val.label,
+            value: val.id,
+            icon: val.icon,
+          }))}
+        />
+        <div className={styles.paymentHelp}>
+          <HelpOutline className={styles.helpIcon} />
+          <div className={styles.paymentText}>
+            <Text t="body3">
+              {t`Currently, Carbonmark only accepts Polygon USDC Payments.`}{" "}
+              <Anchor>{t`Learn how
+            to acquire USDC on Polygon.`}</Anchor>
+            </Text>
+            <Text t="body3">
+              {t`If you’d like to retire this project with another form of cryptocurrency, 
+            or with a credit card, you can do so at`}{" "}
+              <Anchor href={urls.app}>app.klimadao.finance.</Anchor>
+            </Text>
+          </div>
+        </div>
+      </div>
+      <InputField
+        id="price"
+        inputProps={{
+          type: "hidden",
+          max: Number(props.balance || "0"),
```

Comment:
I think the ...register() returns the `max` prop anyways so this can be removed

Reply:
done